Get Started/colors.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Technoracy | Colors</title>
+    <link rel="stylesheet" href="../Css/colors.css">
+    <link rel="stylesheet" href="../../Css/navbar.css">
```

Comment:
just a suggestion instead of adding all links in head tag import them in to single file and use it here